pandas/tests/frame/test_apply.py

```diff
@@ -318,6 +318,11 @@ def test_apply_reduce_Series(self, float_frame):
         result = float_frame.apply(np.mean, axis=1)
         assert_series_equal(result, expected)
 
+    def test_apply_reduce_rows_to_dict(self):
+        data = pd.DataFrame([[1, 2], [3, 4]])
+        expected = pd.Series([{0: 1, 1: 3}, {0: 2, 1: 4}])
+        assert_series_equal(data.apply(dict), expected)
```

Comment:
Also minor nit but let's do `result = data.apply(dict)` on previous line then just do `assert_series_equal(result, expected)` here. The latter is consistent with most other tests